multiqc.wdl

```diff
@@ -86,6 +88,8 @@ task MultiQC {
     }
 
     runtime {
+        cpu: cores
```

Comment:
This will crash, because cores is on optional value. But `cpu` can never be `None`. See comment above on how to fix this.